grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/log/LogInterceptor.java

```diff
@@ -0,0 +1,22 @@
+package org.lognet.springboot.grpc.log;
+
+import io.grpc.Metadata;
+import io.grpc.ServerCall;
+import io.grpc.ServerCallHandler;
+import io.grpc.ServerInterceptor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LogInterceptor implements ServerInterceptor {
```

Comment:
There is an example but LogInterceptor was not a part of grpc-spring-boot-starter. I can add it in a separate PR if needed

Reply:
Why  is this class needed ?